src/doc/book/traits.md

```diff
@@ -275,7 +275,7 @@ won’t have its methods:
 [write]: ../std/io/trait.Write.html
 
 ```rust,ignore
-let mut f = std::fs::File::open("foo.txt").expect("Couldn’t open foo.txt");
+let mut f = std::fs::File::create("foo.txt").expect("Couldn’t open foo.txt");
```

Comment:
I think @steveklabnik is suggesting `"Couldn’t open foo.txt"` should be changed to `"Couldn’t create foo.txt"`